content/applications/hr/attendances/hardware.rst

```diff
@@ -2,56 +2,107 @@
 Hardware
 ========
 
-Kiosk management
-================
+Employees who are **not** database users, and therefore do not have access to the **Attendances**
+app, must sign in and out of work using a kiosk. The following are the physical requirements for
+setting up a kiosk.
 
-A kiosk is a self-service station that allows employees to check in and check out for work shifts.
+Kiosk devices
+=============
 
-There are two ways to set up a kiosk:
+A kiosk is a self-service station where employees can check in and out of work with either a
+:ref:`badge <attendances/hardware/badges>` or an :ref:`RFID key fob <attendances/hardware/rfid>`.
+Typically, these devices are dedicated as kiosks only, but any device with an internet browser is
+able to be set up as a kiosk.
 
-- **Laptop and desktop PC**
+A kiosk is used by navigating to the webpage specified in the :ref:`configuration
+<attendances/kiosk-settings>` section of the *Attendances* app.
 
-  Running a kiosk in a web browser is the cheapest and most flexible option. You can print employee
-  badges with any thermal or inkjet printer compatible with your web browser.
+Kiosks are set up using one of the following types of devices:
 
-- **Tablet and mobile phone (Android or iOS)**
+- Laptop PC
+- Desktop PC
```

Comment:
To avoid the term "PC" which is traditionally associated with only Windows machines :)
```suggestion
- - Laptop or Desktop computer
```